app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
+
+<p>You can confirm your account email through the link below:</p>
+
+<p><%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) %></p>
```

Comment:
- [OPTIONAL] Great work so far 👏. However, when I try to **_signup_**, I noticed that your application doesn't display the confirmation page or give the information that users should check their email for the confirmation link. I have to manually get the link from the terminal to confirm and perform email confirmation. 

This is accessible to me as a developer but not as a user. As a user, I would not be able to access the confirmation link because I would be using the browser and not the terminal. This could frustrate users and eventually mean they would not be able to use your application. I would advise you to improve the user experience of your application by making it possible to display the page that directs users to check their mail for the confirmation link and also to deliver the mail with the confirmation link. You can read this [article](https://www.bogotobogo.com/RubyOnRails/RubyOnRails_Devise_Authentication_Sending_Confirmation_Email.php) to learn how to configure such. 👍 😃